src/python/pants/backend/jvm/tasks/jvm_compile/class_not_found_error_patterns.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import re
+
+
+_CLASS_NOT_FOUND_ERROR_PATTERNS = [
```

Comment:
not crazy at all! since the patterns aren't going to be complete, versions of compiler may change, it's more flexible to allow override from config. 

changed.